scripts/cassandra-integration-test.sh

```diff
@@ -2,29 +2,13 @@
 
 set -ex
 
-# Clean up before starting.
-docker rm -f cassandra || true
-docker rm -f cassandra2 || true
-docker network rm integration_test || true
-
-# Create a network so that the schema container can communicate with the cassandra containers.
-docker network create integration_test
-
-# Start cassandra containers whose ports are exposed to localhost to facilitate testing.
-docker run -d --name cassandra --network integration_test -p 9042:9042 -p 9160:9160 cassandra:3.9
-docker run -d --name cassandra2 --network integration_test -p 9043:9042 -p 9161:9160 cassandra:3.9
-
 # Build the schema container and run it rather than using the existing container in Docker Hub since that
 # requires this current build to succeed before this test can use it; chicken and egg problem.
 docker build -t jaeger-cassandra-schema-integration-test plugin/storage/cassandra/
-docker run --network integration_test -e CQLSH_HOST=cassandra -e TEMPLATE=/cassandra-schema/v001.cql.tmpl jaeger-cassandra-schema-integration-test
-docker run --network integration_test -e CQLSH_HOST=cassandra2 -e TEMPLATE=/cassandra-schema/v002.cql.tmpl jaeger-cassandra-schema-integration-test
+docker run -e CQLSH_HOST=localhost -e CQLSH_PORT=9042 -e TEMPLATE=/cassandra-schema/$1.cql.tmpl --network=host jaeger-cassandra-schema-integration-test
+docker run -e CQLSH_HOST=localhost -e CQLSH_PORT=9043 -e TEMPLATE=/cassandra-schema/$2.cql.tmpl --network=host jaeger-cassandra-schema-integration-test
+docker run -e CQLSH_HOST=localhost -e CQLSH_PORT=9044 -e TEMPLATE=/cassandra-schema/$3.cql.tmpl --network=host jaeger-cassandra-schema-integration-test
```

Comment:
where would this container at 9044 be used?  The integration test only seems to use the first two

https://github.com/jaegertracing/jaeger/blob/ea1ac8f85342f6efebce42c0424fa7681fbd9a27/plugin/storage/integration/cassandra_test.go#L90

Reply:
This is added to validate that v003.cql.tmpl applies properly. I thought it was missing hence thought to add.

Reply:
I think we should revisit how we test, are we going to test old schema versions forever?

Reply:
In my opinion, unless we deprecate any schema, we should test it.

Reply:
Well, the old schemas *are* deprecated. It's been almost a year since v3 was implemented, we don't need to support people running on old schemas forever.

Reply:
Do you think we should just test for v003.cql.tmpl and v004.cql.tmpl in that case.

Reply:
At this point, yes.